plugins/modules/mso_tenant_site.py

```diff
@@ -261,6 +261,8 @@ def main():
     # Get tenant_id and site_id
     tenant_id = mso.lookup_tenant(module.params.get("tenant"))
```

Comment:
Do we have similar behaviour for this lookup method and potentially other lookup methods?

Reply:
Added ignore_404 parameter to the lookup methods.